apps/emqx_resource/src/emqx_resource.erl

```diff
@@ -50,6 +50,7 @@
     remove_local/1,
     reset_metrics/1,
     reset_metrics_local/1,
+    reset_metrics_local/2,
```

Comment:
question: why this change ?

Reply:
I can only find apps/emqx_resource/src/proto/emqx_resource_proto_v2.erl which calls the /1 function.